sdk/schemaregistry/tests.yml

```diff
@@ -6,6 +6,8 @@ jobs:
       ServiceDirectory: schemaregistry
       TestTimeoutInMinutes: 300
       BuildTargetingString: azure-schemaregistry*
+      AllocateResourceGroup: false
```

Comment:
Boolean value used here, whereas the conditional checks string. I think pipelines might cast this so it won't matter, but good to be consistent.